### PR TITLE
Ewlj 77 podcast detail prototype

### PR DESCRIPTION
**Pull request by aCyborg**
## Ticket(s)

**Github Issue**

N/A

**Jira Ticket**

- [EWLj-77: Podcast Detail Prototype](https://issues.ama-assn.org/browse/EWLJ-77)

## Description

Create a podcast detail page. This includes:
* audio player atom

## To Test

- [ ] Spin up Patternlab
- [ ] Review the Audio Player Molecule
- [ ] Review the Podcast Page

## Visual Regressions
N/A

## Relevant Screenshots/GIFs
N/A

## Remaining Tasks
N/A

## Additional Notes
N/A

### CODE REVIEW

**Comment by aCyborg** [comment]
Note: This shouldn't be merged until articles is. Might want to hold off the review until then as well to clear out some of the commits from it.

**Review by gleroux02 — APPROVED** [review]
👍 this looks good. I'll merge once the article one is done.

**Pull request merged**
